src/data/local_storage.js

```diff
@@ -133,7 +144,7 @@ p5.prototype.getItem = function(key) {
   let value = localStorage.getItem(key);
   const type = localStorage.getItem(`${key}p5TypeID`);
   if (typeof type === 'undefined') {
-    console.log(
+    console.error(
```

Comment:
we typically avoid using `console.error` so that the code can keep running with a warning in the console

Reply:
Corrected, thanks!

Reply:
looks great!